src/Selenium2Driver.php

```diff
@@ -131,7 +131,7 @@ public function setDesiredCapabilities($desiredCapabilities = null)
         // See https://sites.google.com/a/chromium.org/chromedriver/capabilities
         if (isset($desiredCapabilities['chrome'])) {
 
-            $chromeOptions = array();
+            $chromeOptions = (isset($desiredCapabilities['chromeOptions']) && is_array($desiredCapabilities['chromeOptions']))? $desiredCapabilities['chromeOptions']:array();
```

Comment:
That adds another layer of complexity, because there is already `$desiredCapabilities['chrome']` array with all chrome-specific options.

Reply:
Another way is to use the `array_merge` at the line https://github.com/minkphp/MinkSelenium2Driver/blob/master/src/Selenium2Driver.php#L145
for not erase the content.

The option `w3c` must be set into the `chromeOptions` for disable the W3C WebDriver dialect for ChromeDriver.